src/widgets/edit-patient-details-button.component.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { ConfigurableLink } from '@openmrs/esm-framework';
+import { useTranslation } from 'react-i18next';
+
+export default function EditPatientDetailsButton({ patientUuid }) {
+  const { t } = useTranslation();
+  return (
+    <li className="bx--overflow-menu-options__option">
+      <ConfigurableLink
+        to={`\${openmrsSpaBase}/patient/${patientUuid}/edit`}
```

Comment:
escaping the first `$` is probably not what you wanted here.

Reply:
Actually, it is. This is an unfortunate choice for template variables, which are identical to the ones from JS.

Reply:
(but truthfully, I think you both are right. One could also use the right global variables and actually circumvent the templating resolution)

Reply:
@nmalyschkin @FlorianRappl the another way I can put the variable is:
`"#{openmrsSpaBase}/patient/" + patientUuid + "/edit"`
Should I go with this?

Reply:
Hm not sure if I can follow. what is `#{openmrsSpaBase}`? Why use string concatenation instead of templating?

Reply:
Ohh sorry @FlorianRappl , it's a typing error, its `${openmrsSpaBase}`. 

I am just extending a suggestion to @nmalyschkin, but since you are saying its fine the way it is, I think this conversation is resolved.